src/benchmark.cr

```diff
@@ -1,3 +1,50 @@
+# A Performance Benchmarking Library
+# Overview
+#
+# The Benchmark module provides methods for benchmarking Crystal code, giving
+# detailed reports on the time taken for each task.
+#
+# * Measure the time to construct the string given by the expression
+# ```
+# "a"*1_000_000_000
+# ```
+# 
+# ```
+# require "benchmark"
+#
+# puts Benchmark.measure { "a"*1_000_000_000 }
+# ```
+#
+# On my machine (OSX 10.10.3 on i7 2.2 Ghz) this generates:
```

Comment:
Will do - thanks!


On 20 Jun 2015, 9:42 PM +0200, Jonne Haß<notifications@github.com>, wrote:
>  
> Insrc/benchmark.cr(https://github.com/manastech/crystal/pull/833#discussion_r32887304):
>  
> >+#>+# The Benchmark module provides methods for benchmarking Crystal code, giving>+# detailed reports on the time taken for each task.>+#>+# * Measure the time to construct the string given by the expression>+# ```>+# "a"*1_000_000_000>+# ```>+#>+# ```>+# require "benchmark">+#>+# puts Benchmark.measure { "a"*1_000_000_000 }>+# ```>+#>+# On my machine (OSX 10.10.3 on i7 2.2 Ghz) this generates:
>  
> I think it's good to use a more formal style in documentation instead of a personal experience style, so "For example on a i7 2.2 Ghz running OSX 10.10.3 this generates:" or even leaving that detail out, "This generates the following output:"
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/manastech/crystal/pull/833/files#r32887304).
>  
>  
>